WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/HoloLens/MixedRealityCapture.cs

```diff
@@ -76,7 +81,7 @@ public partial class DevicePortal
         /// Removes a Mixed Reality Capture file from the device's local storage.
         /// </summary>
         /// <param name="fileName">The name of the file to be deleted.</param>
-        /// <returns>Task tracking the deletion request</returns>
```

Comment:
Doesn't StyleCop still complain about these not having a returns element? Even though it's just a Task?